static/js/components/AnonymousMenu.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+/* global SETTINGS:false */
+import React, { Fragment } from "react"
+
+import MixedLink from "./MixedLink"
+import { routes } from "../lib/urls"
+
+type Props = {
+  useScreenOverlay: boolean
+}
+
+const overlayListItemProps = {
+  "data-toggle": "collapse",
+  "data-target": "#nav"
+}
+
+const AnonymousMenu = ({ useScreenOverlay }: Props) => {
+  /* eslint-disable prefer-const */
+  let listItemProps: Object
+  listItemProps = useScreenOverlay ? overlayListItemProps : null
+  return (
+    <ul className="menu-holder">
+      <li {...listItemProps || {}}>
+        <MixedLink
+          dest={routes.login.begin}
+          className="simple"
+          aria-label="Login"
+        >
+          Sign In
+        </MixedLink>
+      </li>
+      <li {...listItemProps || {}}>
+        <MixedLink
+          dest={routes.register.begin}
+          className="simple button"
+          aria-label="Login"
```

Comment:
```suggestion
          aria-label="Create Account"
```

Reply:
See also #154

Reply:
@pdpinch I have requested @arslanashraf7 to rebase his PR after my PR gets merged